app/static/common/empty-no-commit.svg

```diff
@@ -1 +1,24 @@
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" id="no-commit" viewBox="0 0 288 138">
+    <title>
+        No commit selected
+    </title>
+    <style type="text/css">
+        <![CDATA[
+            .outline1 {
+                fill: #cadef3;
+            }
+            .shadow {
+                fill: #ebf2fa;
```

Comment:
I think we can just use the default color here and drop the opacity of the shadow to 50%